### PR TITLE
metrics: use jsonnet to generate resource control panel

### PR DESCRIPTION
**Pull request by nolouch**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/56319

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)

image

- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by JmPotato — APPROVED** [review]

File: pkg/metrics/grafana/tidb_resource_control.jsonnet [hunk 1]
```diff
@@ -0,0 +1,1153 @@
...
+// A new dashboard
+// Add the template variables
+local TiDBResourceControlDash = dashboard.new(
+  title="Test-Cluster-TiDB-Resource-Control",
+  editable=true,
+  graphTooltip="shared_crosshair",
+  refresh="30s",
+  time_from="now-1h",
+)
+                                .addInput(
```
Comment (glorv) [inline pkg/metrics/grafana/tidb_resource_control.jsonnet:36]: Need format?

File: pkg/metrics/grafana/tidb_resource_control.jsonnet [hunk 2]
```diff
@@ -0,0 +1,1153 @@
...
+  )
+).addTarget(
+  prometheus.target(
+    'sum(rate(resource_manager_resource_unit_read_request_unit_sum{k8s_cluster="$k8s_cluster", tidb_cluster="$tidb_cluster", type=~"background"}[1m])) + sum(rate(resource_manager_resource_unit_write_request_unit_sum{k8s_cluster="$k8s_cluster", tidb_cluster="$tidb_cluster", type=~"background"}[1m]))',
+    legendFormat="total",
+  )
+);
+
+// Background Task Resource Utilization
+
```
Comment (glorv) [inline pkg/metrics/grafana/tidb_resource_control.jsonnet:820]: useless blank line

File: pkg/metrics/grafana/tidb_resource_control.jsonnet (hunk 1 above)
Comment (nolouch) [reply pkg/metrics/grafana/tidb_resource_control.jsonnet:36]: already formatted. It's strange after formatting.

**Review by glorv — APPROVED** [review]

**Pull request merged**